cli/src/pcluster/validators/ebs_validators.py

```diff
@@ -31,6 +31,8 @@
 
 EBS_VOLUME_TYPE_TO_IOPS_RATIO = {"io1": 50, "io2": 1000, "gp3": 500}
 
+EBS_VOLUME_TYPE_NOT_SUPPORT_IOPS = ["gp2", "sc1", "st1", "standard"]
```

Comment:
We don't need this new variable you can use the `EBS_VOLUME_TYPE_TO_IOPS_RATIO`

You can simply do:
```python
if volume_type not in EBS_VOLUME_TYPE_TO_IOPS_RATIO:
  self._add_failure...
```

Reply:
Here I am consider the case that if EC2 announce a new volume type that support IOPS, we don't want to block customers by saying the new volume type not support IOPS.

Reply:
If they are going to announce a new volume type anyway our validators are out-dated because `EBS_VOLUME_TYPE_TO_IOPS_RATIO` would be incomplete.

They could suppress the validator if they want to try a volume type not yet supported.

Reply:
Changed